src/lib/utils/createSendEventForHits.ts

```diff
@@ -1,5 +1,5 @@
 import { InstantSearch, Hit } from '../../types';
-import { InsightsEvent } from '../../middleware/insights';
+import { InsightsEvent } from '../../middlewares/insights';
```

Comment:
how come this change is in this PR and not a previous one?

Reply:
I don't remember well but I guess it must be one of the typical mistakes that happens with stacked PRs.